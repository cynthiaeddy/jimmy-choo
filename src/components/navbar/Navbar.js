import React from 'react';
import { Link } from 'react-router-dom';
import ToggleButton from '../navbar/ToggleButton'
import '../../stylesheets/Navbar.css'




const Navbar = () => {
  return (

		<nav id="navbar">
				<Link to="/navbar"><h3>Menu</h3></Link>
			<Link to='/about-james'><h3>James Oseland</h3></Link>




		</nav>
  );
};

export default Navbar;
