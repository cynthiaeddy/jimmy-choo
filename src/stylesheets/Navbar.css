#navbarHome,
#navbar,
#navbarMexico,
#navbarMexicoMobile,
#navbarHomeMobile {
	position: fixed;
	display: flex;
	width: 100vw;
	height: 120px;
	top: 0;
	justify-content: space-between;
	align-items: center;
	padding: 0 10vw;
	margin-top: 50px;
}
#navbarMexicoMobile.jimMobile {
	height: 160px;
	z-index: 5;
}

#navbar,
#navbarMexico {
	padding: 0 15vw;
	margin: 0;
	padding: 0 18vw;
}
#navbarHome {
	margin-top: 0;
	padding: 0;
	width: 70%;
}
#navbarMexicoMobile {
	/* padding: 0; */
	flex-direction: column;
	margin-top: 0;
	align-items: flex-end;
	text-align: right;
	padding: 0 7.5%;
	height: 200px;
	justify-content: initial;
	background: white;
}

#navbarHomeMobile {
	margin-top: 0;
	flex-direction: column;
	justify-content: space-evenly;
	align-items: flex-start;
	padding: 0 7.5%;
	background: white;
	height: 185px;
	padding-top: 10px;
	z-index: 10;
}
#navbarHomeMobile h1 {
	font-size: 50px;
	align-self: flex-end;
}
#navbarHomeMobile h3 {
	margin-bottom: -5px;
}
#navbarHomeMobile h6,
#navbarMexicoMobile h6 {
	padding-top: 0;
	line-height: 1;
	margin-top: 20px;
	margin-bottom: 25px;
}

#navbarSocial {
	position: fixed;
	display: flex;
	width: 14vw;
	height: 100vh;
	top: 0;
	justify-content: space-between;
	align-items: center;
	right: 0;
}
.social {
	display: flex;
	flex-direction: column;
}

a#menu {
	align-self: flex-start;
}
#menu h6 {
	padding-top: 15px;
	align-self: flex-start;
}

.topnav {
	width: 90%;
	display: flex;
	justify-content: space-between;
}

.buyNav {
	display: flex;

	justify-content: space-around;
	font-family: 'EB Garamond', serif;
	font-weight: 500;
	font-style: italic;
	letter-spacing: 2px;
	font-size: 15px;
	margin-bottom: 10px;
	margin-top: 40px;
}
.buyNav.mexico {
	margin-top: 0;
	justify-content: left;
	align-self: flex-end;
}

span {
	font-family: MundoSansPro-Medium;

	font-style: normal;
	letter-spacing: 0px;
	font-weight: 500;
}
h6 {
	font-size: 20px;
	font-family: 'EB Garamond', serif;
	font-weight: 500;
	font-style: italic;
	letter-spacing: 5px;
	line-height: 4;
	padding-top: 28px;
}

h4 {
	font-size: 22px;
	font-family: 'EB Garamond', serif;
	font-weight: 500;
	font-style: italic;
	letter-spacing: 5px;
	line-height: 3;
}
a {
	text-decoration: none;
	color: #000;
}
a:hover {
	color: grey;
}

.toggleNavBar {
	display: flex;
	flex-direction: column;
	justify-content: center;
	align-items: center;
	width: 100vw;
	height: 100vh;
	background: grey;
	border: 1px solid black;
	position: absolute;
	z-index: 5;
	top: 0;
	left: 0;
	margin: 0;
	padding: 0;
}

li {
	display: inline-flex;
}

.toggleNavBar a {
	margin: 20px 0;
	color: white;
}
.toggleNavBar a:hover {
	color: black;
}
.toggleNavBar {
	display: flex;
	flex-direction: column;
	justify-content: center;
	align-items: center;
	width: 100vw;
	height: 100vh;
	background: grey;
	border: 1px solid black;
	position: absolute;
	z-index: 5;
	top: 0;
	left: 0;
	margin: 0;
}

li {
	display: inline-flex;
}

.toggleNavBar a {
	margin: 20px 0;
	color: white;
	text-align: center;
}
.toggleNavBar a:hover {
	color: black;
}

@media (max-width: 424px) {
	.toggleNavBar a {
		font-size: 18px;
	}
}
