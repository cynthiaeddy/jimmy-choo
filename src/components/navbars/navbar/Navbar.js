import React from 'react';
import { BrowserRouter, Link } from 'react-router-dom';
import '../../../stylesheets/Navbar.css'


const Navbar = () => {
  return (

		<nav id="navbar">
			    <BrowserRouter>
				<Link to="/navbar"><h6>Menu</h6></Link>
				</BrowserRouter>
				<h6>James Oselandâ€™s <span class='nav-world'>WORLD FOOD</span></h6>
		</nav>
  );
};

export default Navbar;
