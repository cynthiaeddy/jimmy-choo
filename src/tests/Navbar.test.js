import React from 'react';
import Navbar from '../components/navbars/navbar/Navbar';
import { create } from 'react-test-renderer'

describe('My first snapshot test',()=>{
    test('testing link button', () => {
    let tree = create(<Navbar />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})
