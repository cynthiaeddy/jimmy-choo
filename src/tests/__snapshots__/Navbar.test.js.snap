// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first snapshot test testing link button 1`] = `
<nav
  id="navbar"
>
  <a
    href="/navbar"
    onClick={[Function]}
  >
    <h6>
      Menu
    </h6>
  </a>
  <h6>
    James Oselandâ€™s 
    <span
      class="nav-world"
    >
      WORLD FOOD
    </span>
  </h6>
</nav>
`;
